test/kb_mash_server_test.py

```diff
@@ -147,6 +147,17 @@ def test_mash_search(self):
                   'search_db':'NCBI_Refseq', 'n_max_results':10}
         self.getImpl().run_mash_dist_search(self.getContext(), params)
 
+    def test_mash_search_with_caching(self):
+        ws_name = self.getWsName()
+        params = {
+            'input_upa': self.get_genome_ref(ws_name),
+            'workspace_name': ws_name,
+            'search_db':'NCBI_Refseq',
+            'n_max_results':10,
+            'cache': 1
+        }
+        ret = self.getImpl().run_mash_dist_search(self.getContext(), params)
```

Comment:
Should check that ret is what's expected

Reply:
I thought the same, but was too lazy to do it last night. I think all the info gets dumped into the report object so unfortunately doing a real check is a little tricky. I can add in some basic stuff